dozer/cogs/info.py

```diff
@@ -10,48 +12,85 @@
 
 class Info(Cog):
     """Commands for getting information about people and things on Discord."""
+    datetime_format = '%Y-%m-%d %H:%M:%S UTC'
 
+    @command(aliases=['user', 'memberinfo', 'userinfo'])
     @guild_only()
-    @cooldown(1, 10, BucketType.channel)
-    @command(aliases=['user', 'userinfo', 'memberinfo'])
-    async def member(self, ctx, member: discord.Member = None):
-        """
-        Retrieve information about a member of the guild.
-        If no arguments are passed, information about the author is used.
-        **This command works without mentions.** Remove the '@' before your mention so you don't ping the person unnecessarily.
-        You can pick a member by:
-        - Username (`cooldude`)
-        - Username and discriminator (`cooldude#1234`)
-        - ID (`326749693969301506`)
-        - Nickname - must be exact and is case-sensitive (`"Mr. Cool Dude III | Team 1234"`)
-        - Mention (not recommended) (`@Mr Cool Dude III | Team 1234`)
```

Comment:
This information seems reasonable to keep as-is or move to the example usage. Anecdotally, `&user` seems to be used most often with mentions, and pingsock/angryping type responses are fairly frequent.

Reply:
Ok